types/wlr_output.c

```diff
@@ -448,15 +448,16 @@ bool wlr_output_attach_render(struct wlr_output *output, int *buffer_age) {
 
 bool wlr_output_preferred_read_format(struct wlr_output *output,
```

Comment:
Is this guaranteed to be called while no other output is attached? Could this cause problems when another output is attached?

Reply:
Yeah, that's a problem, but already was before. I'll open a follow-up issue about this.

Reply:
https://github.com/swaywm/wlroots/issues/2296